app/src/main/java/ar/com/wolox/android/cookbook/twitterlogin/TwitterLoginRecipeActivity.kt

```diff
@@ -0,0 +1,39 @@
+package ar.com.wolox.android.cookbook.twitterlogin
+
+import android.content.Intent
+import ar.com.wolox.android.cookbook.R
+import ar.com.wolox.wolmo.core.activity.WolmoActivity
+
+/**
+ * To connect app with TwitterApi:
+ * 1. Register app in "https://developer.twitter.com/en/apps"
+ * 2. Required fields (to obtain all data from api, like user email):
+ * "Callback URL" (complete with "twittersdk://", "Terms of service URL" and "Privacy policy URL"
+ * 3. App permissions: Enable "Request email address"
+ * 4. Save "API Key" and "API Secret Key" from keys and tokens view
+ * 5. Initialize TwitterApp in application (see "Note" at bottom)
+ */
+class TwitterLoginRecipeActivity : WolmoActivity() {
+
+    override fun layout(): Int = R.layout.activity_base
+
+    override fun init() = replaceFragment(R.id.vActivityBaseContent, TwitterLoginRecipeFragment())
+
+    override fun onActivityResult(requestCode: Int, resultCode: Int, data: Intent?) {
+        super.onActivityResult(requestCode, resultCode, data)
+
+        supportFragmentManager.findFragmentById(R.id.vActivityBaseContent)
+                ?.onActivityResult(requestCode, resultCode, data)
+    }
+}
+
+/**
+ * Note: In WolmoApplication, initialize Twitter App:
+ *
+ * val config = TwitterConfig.Builder(this)
```

Comment:
Remove this commented lines

Reply:
I moved that comments to a javadoc  comment to show how obtain auth token, secret token, etc... from login session.
It's ok or i need to delete it?

Reply:
If you are going to keep this, we should implement it somewhere. What do you think?